src/app/parcels/ParcelsPage.tsx

```diff
@@ -322,9 +322,14 @@ const ParcelsPage: React.FC<{}> = () => {
         lastStatus: rowToLastStatusColumn,
     };
 
-    const onParcelTableRowClick = (row: Row<ParcelsTableRow>): void => {
-        setSelectedParcelId(row.data.parcelId);
-        router.push(`/parcels?${parcelIdParam}=${row.data.parcelId}`);
+    const onParcelTableRowClick = (row: Row<ParcelsTableRow>, event: React.MouseEvent): void => {
+        const target = event.target as HTMLElement;
+        if (target.id.startsWith("cell-1")) {
```

Comment:
I don't think this is specific enough to target the checkbox. It wouldn't be easy to spot this if we decide to show an icon as the first column for example.

Seems like there's a `ignoreRowClick` flag in the DataTable column. Could we use this, and define the checkbox column to ignore row clicks?
https://react-data-table-component.netlify.app/?path=/docs/api-columns--docs

Reply:
Think I need some help on this

Reply:
I realised I can set ignoreRowClick on checkboxconfig